data/data/alibabacloud/cluster/outputs.tf

```diff
@@ -1,3 +1,6 @@
+output "resource_group_id" {
```

Comment:
This should be used as an input into the bootstrap stage. Instead, the bootstrap resources are still using the `ali_resource_group_id` variable.

```
./data/data/alibabacloud/bootstrap/main.tf:98:  resource_group_id = var.ali_resource_group_id
./data/data/alibabacloud/bootstrap/main.tf:133:  resource_group_id = var.ali_resource_group_id
```

Reply:
@bd233 Have you addressed this comment?

Reply:
@kwoodson Yes, I have updated these.